src/nprogress.scss

```diff
@@ -0,0 +1,77 @@
+$nprogress-bar-color: rgb(var(--color-surface-300));
+$nprogress-height : 4px;
```

Comment:
What about replacing `px` with `vh` ?

Reply:
Thats not static. Ever seen any spinner looking responsive?